source/GameData.cpp

```diff
@@ -290,7 +292,14 @@ void GameData::LoadShaders()
 
 double GameData::GetProgress()
 {
-	double spriteProgress = static_cast<double>(spriteLoadingProgress) / totalSprites;
+	double spriteProgress = 0.;
+	if(queuedAllImages)
+	{
+		if(!totalSprites)
+			spriteProgress = 1.;
```

Comment:
```suggestion
			spriteProgress = 0.;
```
I'd expect the situation where totalSprintes is zero to only happen when we start loading, not at any later time. So the progress would be zero, not 1.

Reply:
I think this is for cases where we skip image loading (such as testing).

It is after the `queuedAllImages` check, so it should only set the progress after we are sure there aren't any images.

Reply:
Is it possible that all images are queued, but that none is loaded yet?

Reply:
By the time `queuedAllImages` is set to `true`, `totalSprites` should have reached its final value; if it is zero, that means there were no sprites to load.
I have a special case for that because simply dividing `spritesLoaded` by `totalSprites` in the case of no sprites to load will involve dividing by zero, which is probably something we don't want to do.

Reply:
Ok